docs/source/addons/theme.md

```diff
@@ -169,17 +179,21 @@ From your add-on code, you can extend an existing theme by creating a file corre
 
 ### Variables (`addonsThemeCustomizationsVariables`)
 
-Use this entry point file to modify the original variables of the current loaded theme by adding the entry point after your own variable definitions in the theme.
+Use this entry point file to modify the original variables of the current loaded theme by adding the entry point before the theme variable definitions.
 In the theme, it should be imported as shown below:
 
 ```scss hl_lines="2"
-@import 'variables';
 @import 'addonsThemeCustomizationsVariables';
+@import 'variables';
 @import 'typography';
 @import 'utils';
 @import 'layout';
 ```
 
+```{warning}
+Following SCSS best practices, your theme variables should be "overridable" using the `!default` keyword. Then your customizations should be loaded before them. For more information: https://sass-lang.com/documentation/variables#default-values
```

Comment:
Let's copy-pasta from the authoritative source https://sass-lang.com/documentation/variables#default-values

```suggestion
Following SCSS best practices, your theme variables should be "overridable" using the `!default` flag.
This assigns a value to a variable _only_ if that variable isn't defined or its value is [`null`](https://sass-lang.com/documentation/values/null).
Otherwise, the existing value will be used.
For more information, see https://sass-lang.com/documentation/variables#default-values
```